elastic/kibana
# [Uptime] Default uptime alert type and disable changing type

## Summary

Fixes https://github.com/elastic/kibana/issues/61836.

We had discussed making it so that the Uptime alert is the default UI shown in our alert flyout, and disabling the feature that allows the user to change the type. We even wrote code for this but seem to have forgotten to supply the props necessary for the feature to engage.

This PR is designed to add (remove?) that functionality.

## Testing this PR

1. Open the uptime app
2. Open the alert flyout
3. Observe Uptime alert type by default
4. Try to change the alert type (you should not be able to do this)

### Checklist

Delete any items that are not applicable to this PR.


### For maintainers
